### ISSUE TITLE
[BUG] WXKG12LM - Deconz returns 1007 (and not 1006) when shaken

### ISSUE THREAD

**Issue by cntnry** [xaviml/controllerx#184]
# Bug report

## Description
deCONZ gives "1007" as the event ID when shaking a WXKG12LM controller. ControllerX is configured to expect "1006" with this action. I don't know if this is a mistake or if there has been changes in deCONZ recently?

## Additional information

* Devices involved:
  * Model: WXKG12LMLightController
* Integration: deconz
* AppDaemon version: v4.0.5
* ControllerX version: v4.1.0
* HACS version (if installed from there): v1.6.2
* Home Assistant Core version: v0.118.3
* deCONZ version: v2.05.88

## AppDaemon app configuration

```yaml
soverom_lys:
    module: controllerx
    class: WXKG12LMLightController
    controller: smart_switch
    integration: deconz
    light: light.soverom_lys
```

## Logs

```
{
    "event_type": "deconz_event",
    "data": {
        "id": "smart_switch",
        "unique_id": "00:15:8d:00:02:5d:71:df",
        "event": 1007
    },
    "origin": "LOCAL",
    "time_fired": "2020-11-23T21:47:13.555268+00:00",
    "context": {
        "id": "461a682f3191a936ed86192a1d699521",
        "parent_id": null,
        "user_id": null
    }
}
```

**Comment by xaviml** [comment]
Hi @cntnry,

Thank you for reporting this. This mapping came from @phdelodder from #90. It either changed from deconz or @phdelodder provided the wrong mapping.

I will change this for the next release.

Cheers,
Xavi M.

**Comment by phdelodder** [comment]
Hi, 

I'm receive still a 1006, could this be a new version of the device? 

Kr, 
Philippe

**Comment by cntnry** [comment]
Hi,

Thanks for replying so fast.

Interesting, could be. I have two WXKG12LM and they are both version 20180629. Which version do you have?

Are you using the same version of deCONZ as I am?

BR
Emil

**Comment by phdelodder** [comment]
Indeed it's an error on my side, its indeed 1007.

**Comment by xaviml** [comment]
Thank you both, I will change this for the next release. Hopefully coming next weekend.

In the meantime you can use the "merge_mapping" attribute to add the 1007 action in your configs if you are interested.

Regards,
Xavi M.

**Comment by xaviml** [comment]
This has been fixed in ControllerX [v4.2.0](https://github.com/xaviml/controllerx/releases/tag/v4.2.0).
